package.json

```diff
@@ -291,7 +291,7 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn esbuild ./src/extension.ts --bundle --platform=node --external:vscode --outfile=out/extension.js",
-    "compile": "tsc -p ./",
+    "compile": "tsc -p ./ && mkdir -p out/test/suite/test_submodule/ && cp src/test/suite/test_submodule/.git out/test/suite/test_submodule/",
```

Comment:
should there be a file in this pull request at `src/test/suite/test_submodule/.git`?

Reply:
doh! There should be! I guess it will be better to create it on demand in test rather than somehow forcing it into the repo.

And I didn't update the test… Pushing the fix now!